mitsurunner.yaml

```diff
@@ -59,13 +63,16 @@ globals:
   - id: G_TOPIC_ERROR
     type: const char*
     initial_value: $topic_error
+  - id: G_TOPIC_OUTDOOR_TEMPERATURE_RAW
+    type: const char*
+    initial_value: $topic_outdoor_raw
 
 
 # dallas DS28B20 sensor
 # device measures temperature in 10 seconds interval
 dallas:
   - pin: $dallas_pin
-    update_interval: 10s
+    update_interval: 5s
```

Comment:
One disadvantage of decreasing this is that there will be a lot of data to be collected to databases. If not filtering these out, the grafana get's slow to show longer perioids of temperature data. (when grafana is installed e.g. to RasPi)

In reality, these are not needed to be measured really often in practical use. For debugging and checking defrosting time, this can be short and that is what we are still doing --> This can be like that now.

Reply:
It doesn't increse amount of data, if "send_every" value is 2 or more. You can see below that they are now 6, so there is actually less data going to the database than before. I think that it may be good to sample more often if for example the heat exchanger sensor gives those NaN values so there would be not so big gaps if there are NaN values.

Reply:
Great!